packages/vue/src/StoreDesign/RequestForm/Form/SelectFont.vue

```diff
@@ -45,8 +45,10 @@ if (style.styleSheet) {
         class="p-3 flex items-center text-oc-text-400 justify-between cursor-pointer hover:bg-oc-accent-1-50 gap-x-3 rounded-sm"
         :class="modelValue === option.value ? 'bg-oc-accent-1-50' : ''"
         @click="
-          selectOption(option.value)
-          $emit('update:model-value', option.value)
+          () => {
+            selectOption(option.value)
+            $emit('update:model-value', option.value)
+          }
```

Comment:
does this work? does the 2 functions above get fired?
would it be cleaner replacing this with one function that then calls these 2 functions?